src/main/java/com/google/cloud/tools/managedcloudsdk/components/WindowsBundledPythonCopier.java

```diff
@@ -44,6 +49,32 @@
     // environment passed through via ProcessBuilder will result in cryptic : "The syntax of
     // the command is incorrect."
     String tempPythonLocation = commandCaller.call(copyPythonCommand, null, null).trim();
+
+    Runtime.getRuntime()
+        .addShutdownHook(
+            new Thread() {
```

Comment:
Could reduce this to `new Thread(() -> deleteCopiesPython(tempPythonLocation))`

Reply:
+1